mypy.ini

```diff
@@ -1,44 +1,15 @@
 [mypy]
-python_version=3.7
-
-show_column_numbers=True
-show_error_codes=True
```

Comment:
for instance, these are all false by default:

https://mypy.readthedocs.io/en/stable/config_file.html#confval-show_error_codes

Reply:
and ditto for most of the others

Reply:
Ok, yep - enabling those seems like a good idea.  Why are they disabled by default?